suse_openstack_cloud

```diff
@@ -395,14 +395,14 @@ fi
 ###############################################################
 section_header "RabbitMQ broker"
 
-if [ -e /usr/sbin/rabbitmqctl ]; then
-    rabbitmq_tmpdir=$(mktemp -d)
-    rabbitmq_report_log=$rabbitmq_tmpdir/rabbitmq_report.log
+rabbitmqctl=/usr/lib64/rabbitmq/bin/rabbitmqctl
```

Comment:
Btw, this works right now:

```
type -p rabbitmqctl &> /dev/null
if [ $? -eq 0 ]; then
    rabbitmqctl_report=/var/log/rabbitmq/rabbitmq_report
    su rabbitmq -s /bin/sh -c "/usr/sbin/rabbitmqctl report" > ${rabbitmqctl_report}
    plog_files 0 ${rabbitmqctl_report}
    rm ${rabbitmqctl_report}
fi
```

(we could remove the `su` once @dirkmueller's fix is merged everywhere)